components/bases-locales/validator/report/fields/found-fields.js

```diff
@@ -45,6 +67,23 @@ function FoundFields({fields}) {
           th, td {
             padding: 0.5em;
           }
+
+          .lr-field {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+          }
+
+          .lr-tooltip {
+            width: 125px;
```

Comment:
Jamais trop fan des dimensions fixé dans le marbre. D'autant plus qu'ici, nous aurons de nouvelles langues régionales dont nous ne maitriserons pas la taille.
```suggestion
            width: max-content;
```